INFO     root:Log.py:32 ---------------------------开始执行用例---------------------------------
INFO     root:Log.py:32 用例编号： 1 ,用例名称： 验证手机号码错误，登录失败
INFO     root:Log.py:32 接口请求地址---> http://10.113.248.196/api-user/v1/user/login/phone
INFO     root:Log.py:32 接口请求方式 ---> POST
INFO     root:Log.py:32 接口请求头 ---> {
    "content-type": "application/json",
    "token": "${token}",
    "version": "1.3.0",
    "device": "ios",
    "device-id": "yuanrong-001",
    "device-model": "iphone12",
    "device-os-ver": "13.4.1"
}
INFO     root:Log.py:32 接口请求体 data 参数---> null
INFO     root:Log.py:32 接口请求体 json 参数---> {
    "phone": "123456",
    "smsCode": "123456",
    "deviceId": "test001"
}
INFO     root:Log.py:32 接口请求体 params 参数---> null
INFO     root:Log.py:32 接口上传附件 files 参数---> None
INFO     root:Log.py:32 接口 cookies 参数---> null
DEBUG    urllib3.connectionpool:connectionpool.py:230 Starting new HTTP connection (1): 10.113.248.196:80
DEBUG    urllib3.connectionpool:connectionpool.py:442 http://10.113.248.196:80 "POST /api-user/v1/user/login/phone HTTP/1.1" 200 None